QUEST_LV_0100.tsv

```diff
@@ -6889,7 +6889,7 @@ QUEST_LV_0100_20150511_006884	Varkis' Spirit
 QUEST_LV_0100_20150511_006885	{memo X}That is my adventure journal.. lots of my diaries are hidden at Rukas Plateau besides that..{nl}I want you to retrieve those..
 QUEST_LV_0100_20150511_006886	{memo X}That object is not useful to me anymore now..{nl}If you could find those diaries, I will make you the completed version.
 QUEST_LV_0100_20150511_006887	{memo X}Here it is..{nl}The completed version.. I will leave now..
-QUEST_LV_0100_20150511_006888	The one who fell seeking for knowledge.{nl}Gone back to the Goddess by becoming a part of the canyon.{nl}Saddened mind would penetrate the gold stones, the liquor that is poured would permeate onto the ground for tears..{nl}
+QUEST_LV_0100_20150511_006888	The one who fell while researching..{nl}Gone back to the Goddess by becoming a part of the canyon.{nl}Saddened mind would penetrate the gold stones, the liquor that is poured would permeate onto the ground for tears..{nl}
```

Comment:
The term 'researching' is ambiguous. The original says he is exploring for knowledge.